CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
```

Comment:
I think in the past we've normally added an `Unreleased` section at the top as well

Reply:
Added!